cluster-autoscaler/cloudbuild.yaml

```diff
@@ -1,60 +1,15 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 3600s
+# this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
```

Comment:
micro nit: can we indent this so it has vertical alignment w/ the `substitution_option` property assignment?

Reply:
done